src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue14801.cs

```diff
@@ -0,0 +1,36 @@
+﻿#if ANDROID
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue14801 : _IssuesUITest
+	{
+		public override string Issue => "NullReferenceException in UpdateLeftBarButtonItem";
+
+		public Issue14801(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		[Category(UITestCategories.CarouselView)]
```

Comment:
I don't think this is the right category?

Reply:
Copy paste tricked me! Good catch!
Fixed.